components/LogoAndTitle.jsx

```diff
@@ -61,11 +61,17 @@ const styles = theme => ({
 });
 
 const LogoAndTitle = (props) => {
-  const { classes, timestamp, line } = props;
+  const { classes, timestamp, line, date } = props;
   let defaultText = "";
-  if (line) {
-    defaultText = `How reliable is the ${linesById[line]["name"]} Line today?`
-  };
+  if (line === 'All') {
+    defaultText = date === 'Yesterday' ?
+      'How reliable was the LA Metro Network yesterday?' :
```

Comment:
This might end up being something like:
`How reliable was the LA Metro Network on ${dateFormatted}`